pkg/apiserver/handlers.go

```diff
@@ -378,6 +378,16 @@ func (r *requestAttributeGetter) GetAttribs(req *http.Request) authorizer.Attrib
 	// in empty (does not understand defaulting rules.)
 	attribs.Namespace = apiRequestInfo.Namespace
 
+	// If the raw path is just a single item, the request is to a special resource.
+	if len(apiRequestInfo.Raw) == 1 {
+		switch apiRequestInfo.Raw[0] {
+			case "api":
```

Comment:
I don't want to special case non-resource urls here... what about `/swaggerapi`, `/`, etc? Ideally, policy would be able to address all of those things consistently